app/assets/stylesheets/application.css.scss

```diff
@@ -106,44 +112,82 @@ body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
       border-bottom: 4px solid #f9f9f9;
     }
   }
+
+  ul.side-menu {
+    padding-left: 0;
+  }
   ul, ul a {
     list-style-type: none;
     color: #f9f9f9;
+
   }
   ul span {
     padding-top: 8px;
-    display: block;
+
   }
   li {
-    margin-top: 20px;
-  }
-  .navbar-inverse {
-    background: none;
-    border: none;
-    position: fixed;
-    .navbar-nav {
-      .dropdown-toggle {
-        height: 120px;
-        }
-      .active>a, .active>a:hover, .active>a:focus{
-        background: transparent;
-        font-size: 1.2em;
-        }
-      .open > a, .open>a:hover, .open > a:focus{
-        background: none;
-        height: 140px;
-        transition: 0.5s;
-        z-index: 2;
-      }
+    line-height: 2em;
+    margin-top: 10px;
+    padding: 10px 0 10px 10px;
+    a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3